src/components/emptyState/EmptyState.tsx

```diff
@@ -11,12 +11,15 @@ import theme from './theme.css';
 import { BoxProps } from '../box/Box';
 import { GenericComponent } from '../../@types/types';
 import { SIZES } from '../../constants';
+import BadgedLink, { BadgedLinkProps } from '../badgedLink';
+import { IconAddSmallOutline } from '@teamleader/ui-icons';
 
 export interface EmptyStateProps extends Omit<BoxProps, 'size'> {
   hidePointer?: boolean;
   metaText?: ReactNode | string;
   size?: Exclude<typeof SIZES[number], 'tiny' | 'fullscreen' | 'smallest' | 'hero'>;
   title?: ReactNode | string;
+  action?: Omit<BadgedLinkProps, 'icon' | 'inheric'> & { children: ReactNode };
```

Comment:
you have typo - inheric instead of inherit

Reply:
copied this over from somewhere else, didn't notice it :sweat_smile: